README.fr.md

```diff
@@ -26,74 +26,265 @@ Upsilon est un fork d'Omega, un fork d'Epsilon, l'OS de Numworks tournant sur le
 ## Installation
 
 ### Site web
-Rendez-vous sur le [site d'Upsilon](https://lolocomotive.github.io/Upsilon-website) à la section "Installer". 
+Rendez-vous sur le [site d'Upsilon](https://lolocomotive.github.io/Upsilon-website) à la section "Installer".
 Si votre calculatrice est reconnue, qu'elle contient une version d'Epsilon inférieure à 16 et que votre navigateur accepte WebUSB, la page vous proposera d'installer Upsilon.  
 Ne débranchez votre calculatrice qu'une fois l'installation terminée.
 
 ### Manuelle
 
-Tout d'abord, suivez **la première étape** [ici](https://www.numworks.com/resources/engineering/software/build/), puis :
+ *Vous pouvez vous référer à ce  [site internet](https://www.numworks.com/resources/engineering/software/build/)pour la première étape si vous avez des erreurs*
+
+
+
+### 1. Installation du SDK
+
+<br>
 
 <details>
-  <summary><b>Modèle n0100</b></summary>
+<summary><b>1.1 Linux</b></summary>
+
+<br>
+
+
+<details>
+<summary>Debian ou Ubuntu</summary>
+
+<br>
```

Comment:
Can you please indent this text correctly to make it easier to understand?

Reply:
Done.